### PR TITLE
tiup: add three docs

### PR DESCRIPTION
**Pull request by Joyinqin**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add 3 docs:
- tiup/tiup-command-telemetry.md
- tiup/tiup-component-cluster-destroy.md
- tiup/tiup-component-cluster-audit.md
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/4868
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiup/tiup-command-telemetry.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+TiUP uses the `tiup telemetry` command to control telemetry.
+
+> **Note:**
+>
+> This feature is enabled by default.
+
+## Syntax
+
+```sh
```
Comment (ran-huang) [inline tiup/tiup-command-telemetry.md:29]:
```suggestion
```shell
```
Please use `shell` for all `sh` in this PR.
I've asked Tom to update the Chinese docs.

File: tiup/tiup-component-cluster-audit.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+The `tiup cluster audit` command is used to view commands executed on all clusters in the history and the execution log of each command.
+
+## Syntax
+
+```sh
+tiup cluster audit [audit-id] [flags]
+```
+
+- If you don’t fill in the `[audit-id]`, the table of operation records will be output in reverse chronological order. The first column is the `audit-id`.
```
Comment (ran-huang) [inline tiup/tiup-component-cluster-audit.md:15]:
```suggestion
- If you do not fill in the `[audit-id]`, the table of operation records is output in reverse chronological order. The first column is the `audit-id`.
```

File: tiup/tiup-component-cluster-audit.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+### -h, --help
+
+- Prints the help information.
+- Data type: `Boolean`
+- Default: false
+
+## Outputs
+
+- If `[audit-id]` is specified, the corresponding execution log will be output.
```
Comment (ran-huang) [inline tiup/tiup-component-cluster-audit.md:28]:
```suggestion
- If `[audit-id]` is specified, the corresponding execution log is output.
```
Please use the present tense in this PR.

File: tiup/tiup-component-cluster-destroy.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+---
+title: tiup cluster destroy
+---
+
+# tiup cluster destroy
+
+After the business goes offline, if you want to release the machines occupied by the cluster for use by other businesses, you need to clean up the data on the cluster and the deployed binary files. To destroy the cluster, the `tiup cluster destroy` command performs the following operations:
```
Comment (ran-huang) [inline tiup/tiup-component-cluster-destroy.md:7]:
```suggestion
After an application goes offline, if you want to release the machines occupied by the cluster for use by other applications, you need to clean up the data on the cluster and the deployed binary files. To destroy the cluster, the `tiup cluster destroy` command performs the following operations:
```

File: tiup/tiup-component-cluster-destroy.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+- Stops the cluster.
+- For each service, delete its log directory, deployment directory, and data directory.
+- If the parent directory of the data directory or deployment directory of each service is created by tiup-cluster, also delete it.
```
Comment (ran-huang) [inline tiup/tiup-component-cluster-destroy.md:11]:
```suggestion
- If the parent directory of the data directory or deployment directory of each service is created by tiup-cluster, also delete the parent directory.
```

File: tiup/tiup-component-cluster-destroy.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+## Syntax
+
+```sh
+tiup cluster destroy <cluster-name> [flags]
+```
+
+`<cluster-name>`: the name of the cluster to destroy.
```
Comment (ran-huang) [inline tiup/tiup-component-cluster-destroy.md:19]:
```suggestion
`<cluster-name>`: the name of the cluster to be destroyed.
```

**Comment by ran-huang** [comment]
Please address this~ https://github.com/pingcap/docs/pull/5061#discussion_r604565325

File: tiup/tiup-command-telemetry.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
+---
```
[the remark this answers is not in the document]
Comment (TomShawn) [reply tiup/tiup-command-telemetry.md:1]: After all en TiUP docs in pingcap/docs-cn#4868 is ready, I will create a separate PR to add TOC for all TiUP docs.

**Comment by Joyinqin** [comment]
addressed, PTAL again @ran-huang thx~

**Review by ran-huang — APPROVED** [review]

**Pull request merged**